### ISSUE TITLE
Tooltips don't appear in the scene tree or script list

### ISSUE THREAD

**Issue by pouleyKetchoupp** [godotengine/godot#41262]
**Godot version:**
Master d3b5c0948c943b3fbcb4b5322262c59c92abfa83

**OS/device including version:**
MacOS Catalina
Ubuntu 20.04

**Issue description:**
On Mac or Linux, tooltips don't appear at all in the scene tree or script list.

On Linux, it might be related to #37674 (difficult to validate that it's actually showing behind the window).
On Mac, tooltips in the inspector work properly so it can't be the case.

**Steps to reproduce:**
Case 1. In any scene, hover on a node in the scene tree panel.
Case 2. In the script editor, hover on a script in the list.

**Minimal reproduction project:**
Can be easily reproduced in any project.

**Comment by coppolaemilio** [comment]
I can also mention that on my Windows 10 dual screen setup the tooltips appear on the second monitor and they blink a lot. Not sure if related but probably it is. If it's not clear I can probably make a video of it.

**Comment by pouleyKetchoupp** [comment]
@coppolaemilio There's a separate issue for the blinking tooltips on Windows: #41261

> the tooltips appear on the second monitor

Do you mean they are showing on the wrong monitor while the rest of the editor is on the first monitor?

**Comment by coppolaemilio** [comment]
> 
> 
> @coppolaemilio There's a separate issue for the blinking tooltips on Windows: #41261
> 
> > the tooltips appear on the second monitor
> 
> Do you mean they are showing on the wrong monitor while the rest of the editor is on the first monitor?

Yes.
```
[ Monitor A ]        -       [ Monitor B]
    Godot                       Nothing
hover on element ->          lonely tooltip
```

If I move the editor from **Monitor A** to **Monitor B** the tooltips are not showing up, which I guess it means that they are drawing under the editor.
Another curious thing is that the x and y position of the tooltip correspond to the position where my mouse is in **Monitor A**. So if the tooltip should be in **Monitor A** 120,340 the tooltip shows up on **Monitor B** 120,340.

**Comment by pouleyKetchoupp** [comment]
@coppolaemilio Thanks for the details. I don't have the setup to try and reproduce it, but could you open a new issue for this so we can track it separately? I don't know if it's Windows specific at this point.

**Comment by KoBeWi** [comment]
Is this still reproducible in the current master? Pretty sure it should've been fixed at this point.

**Comment by Calinou** [comment]
Seems fixed on Linux as of 212989d46, even if I move the editor window to another monitor:

![image](https://user-images.githubusercontent.com/180032/166336417-87cc6e01-f9af-44e4-9fb8-9512aa4f8ae4.png)

However, tooltips still fail to appear if **Single Window Mode** is enabled in the Editor Settings. This should be tracked in a separate issue though (maybe there's already one).
